charts/sn-platform/templates/vault/vault-statefulset.yaml

```diff
@@ -98,6 +114,9 @@ spec:
   # See https://www.vaultproject.io/docs/configuration/ for more information.
   config:
   {{ toYaml .Values.vault.config | nindent 4 }}
+  {{- if .Values.vault.securityContext }}
```

Comment:
Is this necessary?

Reply:
Without this config, vault will fail to start and logs as 

```
Error initializing core: Failed to lock memory: cannot allocate memory

This usually means that the mlock syscall is not available.
Vault uses mlock to prevent memory from being swapped to
disk. This requires root privileges as well as a machine
that supports mlock. Please enable mlock on your system or
disable Vault from using it. To disable Vault from using it,
set the `disable_mlock` configuration option in your configuration
file.
2022-08-11T01:26:29.465Z [INFO]  proxy environment: http_proxy="" https_proxy="" no_proxy=""
```

But from the vault docs: https://www.vaultproject.io/docs/configuration#disable_mlock, seems we can use environment variable `VAULT_DISABLE_MLOCK` also.